### ISSUE TITLE
Resize and reposition simliar to v 1.6.2

### ISSUE THREAD

**Issue by sgsweb** [nyroDev/nyroModal#178]
The following code in 1.6.2:

``` javascript
        $("a#popup").click (function () {
           $.fn.nyroModalManual({
            width:400,
            height:200,
            minWidth:400,
            minHeight:200,
            resiable:true,
            content:$("div#popupmsg").html()
            });
          });
```

Similar code in 2.0.0

``` javascript
        $("a#popup").click (function () {
          $.nmData($("div#popupmsg").html(),
          {
            sizes: {w:400,h:200}
          });
          });
```

in version 1.6.2 will allow the modal to reposition and resize automatically as the browser size is changed.  The change in the modal is smooth with animation.  This feature does not seem to be available in the new 2.0.0 version.  If it is, what is the process for recreating this feature in 2.0.0.

**Comment by sgsweb** [comment]
This seems to be an issue for more people now.  Any updates will be greatly appreciated.

**Comment by nyroDev** [comment]
I think the bug has been fixed with commit 0d162edcdfd3d4e47fb7e778f2347ba62b505c97
